README.md

```diff
@@ -170,7 +242,7 @@ native_model = { version = "0.4", features = [ "rmp_serde_1_3" ] }
 
 ```rust
 #[derive(Clone, Default, serde::Deserialize, serde::Serialize)]
-#[native_model(id = 1, version = 1, with = native_model::rmp_serde_1_3::RmpSerde)]
+#[native_model::native_model(id = 1, version = 1, with = native_model::rmp_serde_1_3::RmpSerde)]
```

Comment:
To be consistent with the other examples, is it possible to import the library `use native_model::native_model;` to have?
```suggestion
#[native_model(id = 1, version = 1, with = native_model::rmp_serde_1_3::RmpSerde)]
```

Reply:
Sure, let me quickly update it.

Reply:
Perfect :)